test/mock_relay.cpp

```diff
@@ -644,17 +718,46 @@ TEST(relay, server_callback) {
   config.state_db = state_db;
   config.local_sock = -1;
 
-  // negative case testing
-  try {
-    server_callback(0, 0, &config);
-  }
-  catch (const std::exception& e) {
-    EXPECT_TRUE(false);
-  }
+   // same name to override static global variable
+  uint8_t server_recv_buffer[BUFFER_SIZE] = {
```

Comment:
I don't think you can override a static global variable in another .cpp file. Are you sure this is working?

Reply:
Yes, Saikrishna, you are right. It's some dummy code I tried in my testing, I will remove them for less confusion.

Reply:
Can you remove this code?

Reply:
Sure, my miss, will apply in latest diff.